lib/ipmi_backend_utils.pm

```diff
@@ -128,6 +128,7 @@ sub setup_console_in_grub {
               = "cp $grub_cfg_file ${grub_cfg_file}.org "
               . "\&\& sed -ri 's/timeout=-{0,1}[0-9]{1,}/timeout=30/g' $grub_cfg_file "
               . "\&\& sed -ri 's/^terminal.*\$/terminal_input console serial\\nterminal_output console serial\\nterminal console serial/g' $grub_cfg_file "
+              . "\&\& sed -ri 's/console=.* /console=$ipmi_console,115200 console=tty loglevel=5 /g' $grub_cfg_file"
```

Comment:
This is already defined the same in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/ipmi_backend_utils.pm#L118 as well as https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/proxymode.pm#L156
I suggest to not set it in a third place. Better find a common way to define in a single location and use from there.

Reply:
I can take this into consideration, but can not promise  breakthrough innovation. It depends on their usage.

Reply:
Changes have been made.